lib/attr_json/record.rb

```diff
@@ -161,6 +161,33 @@ def attr_json(name, type, **options)
         # it with usual system will let simple_form and maybe others find it.
         if options[:rails_attribute]
           self.attribute name.to_sym, self.attr_json_registry.fetch(name).type
+
+          # Ensure the `attributes` are set to container attribute values after a
```

Comment:
It's not actually ensuring they are set to _container_ attributes, it's ensuring attributes are set for our actual attr_json values, right?